### DISCUSSION TITLE
Threshold of the score for interactions

### DISCUSSION THREAD

**Discussion by hqdream** [chaidiscovery/chai-lab#135] [Q&A] [answered]
The chai-1 provides an excellent experience for molecule interaction prediction at a very fast speed. It is very useful that the chai-1 gives a final score for each prediction. I understand that the score is an aggregate score including several scores like ptm, iptm, per_chain_p, etc. However, I am wondering if there is a threshold of the score that is easier to tell the possibility of interaction, like the DocQ scores in which <0.23 incorrect, 0.23-0.49 acceptable, 0.49-0.8 medium, >0.8 high. That will be very helpful when plenty of predictions need to be analyzed. Thanks a lot.

**Comment by MattMcPartlon** [comment, marked as answer]
We don't have any strict cutoffs for interaction prediction. I'd recommend calibrating this based on known binders/non-binders - i.e. find some binders/non-binders in the literature and run inference on them with Chai1 then determine confidence-score thresholds that best discriminate between the two groups.
